Cargo.toml

```diff
@@ -1,38 +1,88 @@
+# Copyright (C) 2022 Nicolas Stalder AND  Nitrokey GmbH
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
 [package]
 name = "piv-authenticator"
-version = "0.0.0-unreleased"
-authors = ["Nicolas Stalder <n@stalder.io>"]
-edition = "2018"
+version = "0.3.2"
+authors = ["Nicolas Stalder <n@stalder.io>", "Nitrokey GmbH"]
+edition = "2021"
 license = "Apache-2.0 OR MIT"
-repository = "https://github.com/solokeys/piv-authenticator"
+repository = "https://github.com/nitrokey/piv-authenticator"
 documentation = "https://docs.rs/piv-authenticator"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[example]]
+name = "vpicc"
+required-features = ["vpicc"]
+
+
+[[example]]
+name = "usbip"
+required-features = ["apdu-dispatch", "virt"]
 
 [dependencies]
 apdu-dispatch = { version = "0.1", optional = true }
-delog = "0.1.0"
+delog = { version = "0.1.5", optional = true }
 flexiber = { version = "0.1", features = ["derive", "heapless"] }
 heapless = "0.7"
 hex-literal = "0.3"
-interchange = "0.2.0"
+interchange = "0.2.2"
```

Comment:
As a general matter, I prefer being least prescriptive here, i.e. using the minimal requirement like "1" or "0.2"; and if anything fixing minimum semver requirements in the Cargo.lock file of the final consumer (which is just regularly `cargo update`d). It's the lazy approach, but it seems to work :)

Reply:
There are a lot of people that want to get `-Z minimal-version` to work, so that for example MSVR-aware resolver can work.

For this specific case PIV does not actually depend on `interchange` directly, so I will remove this line.